### DISCUSSION TITLE
Shows dangerous site

### DISCUSSION THREAD

**Discussion by Jitender-verma01** [community/community#122755] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

After live the website chrom shows this is dangerous site 
Others browser run properly

**Comment by LiteBrite82** [comment]
Hi @jitendra-ku-verma 

Can you share a screenshot or a repo link?

This usually happens because of a failed DNS validation or external links in your code that lead to http instead of https

**Reply by Jitender-verma01** [reply]
https://jitendra-ku-verma.github.io/weather_app/

**Reply by Jitender-verma01** [reply]
Every repo can show dangerous site after live

**Reply by LiteBrite82** [reply, marked as answer]
I did some research and testing. 

I verified that your code does work - I cloned your repo and [deployed the application](https://litebrite82.github.io/testing/) 

I did some searching on the issue and found that others have a similar issue when they have created repos and deployments that contain the names of real websites like "netfilx"; which is inline with the messaging you are receiving from the browser. You have a [repo and a deployment](https://github.com/jitendra-ku-verma/Netflix_Homepage) that match that criteria.  

Here is [a brief explanation of the issue](https://github.com/orgs/community/discussions/30261#discussioncomment-3440220) from @airtower-luna 

In the same discussion, further down, @eldadYikne gives [a step by step process to get your page removed from Google's phishing blacklist](https://github.com/orgs/community/discussions/30261#discussioncomment-6026700)

Read through that discussion to get a better understanding of the issue. Its not because of any settings, its because your GH-Pages domain name has been black listed by Google. 

I recommend:
1. Following the instructions to remove your page from the google black list. 
or
2. Delete or rename https://github.com/jitendra-ku-verma/Netflix_Homepage and any references to netflix in your code; including trademarked logos. 
3. [Change your GitHub username](https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-personal-account-on-github/managing-user-account-settings/changing-your-github-username) - though this may not work as the IP address may be blacklisted; not just the domain name. Only do this as a last resort. remember, it may take a few days for Google to respond to your request.

**Reply by Jitender-verma01** [reply]
Problem solved 
Thank you 😊

**Reply by LiteBrite82** [reply]
glad to get verification that changing the username does work 😄

**Reply by sakethpabbathi** [reply]
> Problem solved Thank you 😊

sir .
what u have done to solve this issue

**Comment by ghost** [comment]
hello sir even i too facing the same issue that i gets dangerous site while visiting the browser after published my code in github but write the code by my self doesn't copy that 
![Screenshot (23)](https://github.com/user-attachments/assets/4e4b6cb5-f5f2-4149-8ff5-f6b83b14fd28)
![Screenshot (23)](https://github.com/user-attachments/assets/01894da9-10b5-4494-aab1-51b944e0982c)
please any one solve this

**Reply by BharathiTech7** [reply]
Same problem, 😞 have you got the solution...??

**Comment by Jitender-verma01** [comment]
Chage the repo name if still face prblm then change the GitHub user name

**Comment by fakhrrazi714-ctrl** [comment]
<img width="1080" height="2520" alt="Screenshot_20260809-001004" src="https://github.com/user-attachments/assets/11729ed6-22f0-4dcb-a183-55d5bc7556e9" />
L
<img width="1080" height="2520" alt="Screenshot_20260809-001712" src="https://github.com/user-attachments/assets/ffd7be90-a416-425c-a54c-2ec572e9f56d" />
Live is not working 
<img width="1080" height="2520" alt="Screenshot_20260809-120302" src="https://github.com/user-attachments/assets/282c52ac-d698-4c36-a162-9e62e5176528" />

**Comment by fakhrrazi714-ctrl** [comment]
My live is not working 😭
